float64 target_distance
---
bool distance_exceeded

